sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/protobuf/MetricsImporter.java

```diff
@@ -73,9 +76,18 @@ void consumeFor(InputFile inputFile, MetricsInfo message) {
     }
     saveMetric(context, inputFile, CoreMetrics.NCLOC, message.getCodeLineCount());
 
+    boolean cognitiveComplexityError = message.getCognitiveComplexity() < 0;
+    if(supportsCognitiveComplexity && !cognitiveComplexityError) {
```

Comment:
I guess I just did not know the shortcut to autoformat in IntelliJ (Crtl+Alt+L)...